wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -0,0 +1,107 @@
+# Difficulty Naming
+
+The purpose of difficulty naming is telling players how difficult a map is. This article lists common conventions as well as other alternatives that have been gathered throughout the years. However, as long as the set of difficulty names is descriptive, you may use any desired kind of scheme. 
+
+## Default	
+
+This subsection shows the default **scaling** difficulty names for each game mode, which are the most commonly used.
+
+- ![Marathon](/wiki/shared/diff/expert-s.png "Marathon") **Marathon**: a map that is over 5 minutes of [drain time](/wiki/drain_time), and thus can form a mapset on its own.
```

Comment:
Is X icon necessary here? Or is this just an example? Not all marathons have to be Expert tier difficulty after all, so maybe good to clarify this.